web-client/src/views/FileDocument/ExternalConsolidatedCaseGroupFilingCard.tsx

```diff
@@ -34,7 +34,7 @@ export const ExternalConsolidatedCaseGroupFilingCard = connect(
               >
                 This case is part of a consolidated group.
               </legend>
-              <span className="usa-hint">
+              <span className="usa-hint" style={{display: 'block'}}>
```

Comment:
In general, we have preferred classes to inline styles, although there is certainly work to be done to improve these style classes, and ultimately it will be up to you guys how to handle styling. In any case, I'd suggest using https://github.com/ustaxcourt/ef-cms/blob/894295340b3e8d7bbc2b6b229d831c5446862091/web-client/src/styles/custom.scss#L2026-L2028.

So:

`<span className="usa-hint display-block">`